templates/whatson/event_page.html

```diff
@@ -43,7 +43,17 @@
                                 <i class="fa-solid fa-location-dot event-meta__icon"></i>
 
                                 <div class="event-meta__details">
-                                    <p>{{ page.get_venue_type_display }}</p>
+                                    {% if page.venue_type == "in_person" and page.venue_address %}
+                                        {% if page.venue_website %}
+                                            <a class="event-details-banner__link" href="{{ page.venue_website }}">
+                                        {% endif %}
+                                                {{ page.venue_address|process_address|richtext }}
```

Comment:
Addresses should ideally be wrapped in an `<address>` element

Reply:
done in https://github.com/nationalarchives/ds-wagtail/pull/1467/commits/bf60a0c879ca7d3bdb787285a791045be1c06a85